services/api-gen/src/test/resources/specs/http-asyncapi.yaml

```diff
@@ -0,0 +1,370 @@
+#
+# Copyright 2024 Aklivity Inc
+#
+# Licensed under the Aklivity Community License (the "License"); you may not use
+# this file except in compliance with the License.  You may obtain a copy of the
+# License at
+#
+#   https://www.aklivity.io/aklivity-community-license/
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OF ANY KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations under the License.
+#
+
+---
+asyncapi: 3.0.0
+info:
+  title: API Document for REST APIs
+  version: 1.0.0
+  license:
+    name: Aklivity Community License
+    url: https://github.com/aklivity/zillabase/blob/develop/LICENSE
+servers:
+  http:
+    host: localhost:8080
+    protocol: http
+channels:
+  petstore_customers-stream-identity:
+    address: "/petstore_customers-stream-identity"
```

Comment:
We need to review the generated API paths for HTTP spec to make sure they are as natural as possible.